### DISCUSSION TITLE
FanControl shuts off when computer goes to sleep

### DISCUSSION THREAD

**Discussion by PlayfulDragon** [Rem0o/FanControl.Releases#1717] [General]
I am running v153 on Windows 11.  Initial configuration went great, installed the ADLX plugin, which is working with my Radeon 6800 GPU, and with one exception everything seems to be working as expected.  That one thing is when the computer goes to sleep, the FanControl process stops and does not restart.  As a work around, I modified the startup scheduled task, added unlock as a trigger, with a 10 second delay, and that is getting FanControl restarted.

Is this normal behavior, or should FanControl stay loaded through a sleep cycle?

**Comment by Rem0o** [comment]
It should.  No idea how the ADLX stuff goes through a sleep cycle though. Highly likely it crashes.  It needs a lot of work.

**Comment by niky45** [comment]
Hello, I am having this exact same issue, also have the ADLX plugin, but I should mention the ADLX plugin is doing nothing, i.e. I have told fancontrol to not control the GPU because of this. 

I will try to remove the plugin (since it's doing nothing atm) and see if that makes a difference. 

EDIT: after a quick test (disabled the plugin, got fancontrol to stop whining about missing stuff, saved the config, and put the pc to sleep then woke it up again), it seems that's indeed ADLX that's causing the crash.
